packages/govuk-frontend-review/src/views/examples/template-default/index.njk

```diff
@@ -1,4 +1,4 @@
-{% extends "govuk/template.njk" %}
+{% extends "layouts/examples.njk" %}
```

Comment:
Looking good this @owenatgov 🙌

Probably seen from the tests the [template-default](https://govuk-frontend-pr-4424.herokuapp.com/examples/template-default) example has broken?

https://govuk-frontend-pr-4424.herokuapp.com/examples/template-default

Reply:
Ahaaaaaa that's where my test are going. Cheers Colin I'll sus this out.

Reply:
Thanks @owenatgov, let me know if you need a hand with anything